scripts/partition_manager_report.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+import argparse
+import yaml
+import platform
+
+def print_region(region, size, reqs):
+    # Prepare colors (color code taken from size_report)
+    bcolors_ansi = {
+        "HEADER"    : '\033[95m',
+        "OKBLUE"    : '\033[94m',
+        "OKGREEN"   : '\033[92m',
+        "WARNING"   : '\033[93m',
+        "FAIL"      : '\033[91m',
+        "ENDC"      : '\033[0m',
+        "BOLD"      : '\033[1m',
+        "UNDERLINE" : '\033[4m'
+    }
+    if platform.system() == "Windows":
+        # Set all color codes to empty string on Windows
+        bcolors = dict.fromkeys(bcolors_ansi, '')
+    else:
+        bcolors = bcolors_ansi
+
+    # Print header
+    print(bcolors["OKBLUE"] + "%s (0x%x):" % (region, size) + bcolors["ENDC"])
+
+    # Sort partitions three times:
+    #  1. On whether they are a container (has a 'span'), containers first.
+    #  2. On size, descending.
+    #  3. On address, ascending.
+    sorted_reqs = sorted(sorted(sorted(reqs.keys(), key=lambda x: int('span' in reqs[x]), reverse=True), key=lambda x: reqs[x]['size'], reverse=True), key=lambda x: reqs[x]['address'])
+
+    # Create text lines
+    sizes = ["%s%s (0x%x)" % ((("| " + bcolors["WARNING"] if 'span' not in reqs[part] else "+---" + bcolors["OKBLUE"]) + "0x%x: " % reqs[part]['address']), part, reqs[part]['size']) + bcolors["ENDC"]  for part in sorted_reqs]
```

Comment:
This line is way too long.120 is max per pep8

Reply:
will fix